src/BlobField/index.js

```diff
@@ -105,9 +105,9 @@ export default function BlobField({ collapsed = false }) {
 
   return (
     <div
-      className={style.wrapper}
+      className='blobs-wrapper'
```

Comment:
Again fixing styling when entering the homepage from and external link or by typing in the address bar.